2022-02-19 06:43:24,906 INFO    MainThread:738 [internal.py:wandb_internal():91] W&B internal server running at pid: 738, started at: 2022-02-19 06:43:24.905962
2022-02-19 06:43:24,908 INFO    WriterThread:738 [datastore.py:open_for_write():77] open: /workspace/graph-neural-pde/src/wandb/run-20220219_064324-2m1ok27m/run-2m1ok27m.wandb
2022-02-19 06:43:24,908 DEBUG   HandlerThread:738 [handler.py:handle_request():120] handle_request: check_version
2022-02-19 06:43:24,908 DEBUG   SenderThread:738 [sender.py:send():160] send: header
2022-02-19 06:43:24,909 DEBUG   SenderThread:738 [sender.py:send():160] send: request
2022-02-19 06:43:24,909 DEBUG   SenderThread:738 [sender.py:send_request():169] send_request: check_version
2022-02-19 06:43:25,099 DEBUG   SenderThread:738 [sender.py:send():160] send: run
2022-02-19 06:43:25,645 INFO    SenderThread:738 [sender.py:_start_run_threads():654] run started: 2m1ok27m with start time 1645253004
2022-02-19 06:43:25,646 DEBUG   SenderThread:738 [sender.py:send():160] send: summary
2022-02-19 06:43:25,647 INFO    SenderThread:738 [sender.py:_save_file():781] saving file wandb-summary.json with policy end
2022-02-19 06:43:25,653 DEBUG   HandlerThread:738 [handler.py:handle_request():120] handle_request: run_start
2022-02-19 06:43:26,603 WARNING MainThread:738 [internal.py:wandb_internal():146] Internal process interrupt: 1
2022-02-19 06:43:26,700 DEBUG   HandlerThread:738 [meta.py:__init__():39] meta init
2022-02-19 06:43:26,700 DEBUG   HandlerThread:738 [meta.py:__init__():53] meta init done
2022-02-19 06:43:26,701 DEBUG   HandlerThread:738 [meta.py:probe():210] probe
2022-02-19 06:43:26,701 DEBUG   HandlerThread:738 [meta.py:_setup_git():200] setup git
2022-02-19 06:43:26,713 DEBUG   HandlerThread:738 [meta.py:_setup_git():207] setup git done
2022-02-19 06:43:26,713 DEBUG   HandlerThread:738 [meta.py:_save_pip():57] save pip
2022-02-19 06:43:26,714 DEBUG   HandlerThread:738 [meta.py:_save_pip():71] save pip done
2022-02-19 06:43:26,714 DEBUG   HandlerThread:738 [meta.py:_save_conda():78] save conda
2022-02-19 06:43:26,801 WARNING MainThread:738 [internal.py:wandb_internal():146] Internal process interrupt: 2
2022-02-19 06:43:26,801 ERROR   MainThread:738 [internal.py:wandb_internal():149] Internal process interrupted.
2022-02-19 06:43:26,804 DEBUG   HandlerThread:738 [meta.py:_save_conda():86] save conda done
2022-02-19 06:43:26,804 DEBUG   HandlerThread:738 [meta.py:probe():252] probe done
2022-02-19 06:43:26,806 INFO    HandlerThread:738 [handler.py:finish():624] shutting down handler
2022-02-19 06:43:27,107 INFO    WriterThread:738 [datastore.py:close():258] close: /workspace/graph-neural-pde/src/wandb/run-20220219_064324-2m1ok27m/run-2m1ok27m.wandb
2022-02-19 06:43:27,686 INFO    SenderThread:738 [sender.py:finish():884] shutting down sender
2022-02-19 06:43:30,431 INFO    MainThread:738 [internal.py:handle_exit():78] Internal process exited
